src/v2/Components/useAuctionExtendedBiddingWebsocket.tsx

```diff
@@ -0,0 +1,22 @@
+import { useEffect } from "react"
+import { useWebsocketContext } from "v2/System/WebsocketContext"
+
+interface AuctionExtendedBiddingWebsocketParams {
+  lotID: string
+  onChange: (extendedBiddingEndAt: string) => void
+}
+
+export const useAuctionExtendedBiddingWebsocket = ({
+  onChange,
+  lotID,
+}: AuctionExtendedBiddingWebsocketParams) => {
+  const { data } = useWebsocketContext()
+  const { lot_id, extended_bidding_end_at } = data
+  const receivedMessageForThisLot = lot_id === lotID
+
+  useEffect(() => {
+    if (receivedMessageForThisLot) {
+      onChange(extended_bidding_end_at)
```

Comment:
@mzikherman - if you just pass back the whole data object this hook becomes totally generic. No need to scope it so specifically to extended bidding, its just a prop on `data`.

Reply:
ah yea true, v good call!